EventFilter/CTPPSRawToDigi/plugins/TotemVFATRawToDigi.cc

```diff
@@ -168,9 +169,19 @@ template <typename DigiType>
 void TotemVFATRawToDigi::run(edm::Event &event, const edm::EventSetup &es) {
   // get DAQ mapping
   ESHandle<TotemDAQMapping> mapping = es.getHandle(totemMappingToken);
+  if (!mapping) {
+    LogError("TotemVFATRawToDigi") << "TotemVFATRawToDigi: no mapping found for " << subSystemName;
+  }
 
   // get analysis mask to mask channels
-  ESHandle<TotemAnalysisMask> analysisMask = es.getHandle(analysisMaskToken);
+  TotemAnalysisMask analysisMask;
+  ESHandle<TotemAnalysisMask> analysisMaskHandle = es.getHandle(analysisMaskToken);
+  if (analysisMaskHandle) {
+    analysisMask = *analysisMaskHandle;
+  } else {
+    LogWarning("TotemVFATRawToDigi") << "TotemVFATRawToDigi: no mask found for " << subSystemName;
+    analysisMask = *(std::make_unique<TotemAnalysisMask>());
```

Comment:
```suggestion
    analysisMask = TotemAnalysisMask();
```

try other way of constructing empty mask object

Reply:
done